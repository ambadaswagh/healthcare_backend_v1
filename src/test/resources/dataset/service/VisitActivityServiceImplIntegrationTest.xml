<?xml version="1.0" encoding="UTF-8"?>
<dataset>
    <company id="100"
             license_no="License"
             federal_tax="Federal Tax"
             federal_tax_start="2017-05-01"
             federal_tax_expire="2017-05-20"
             federal_tax_status="1"
             state_tax="State Tax"
             state_tax_start="2017-05-01"
             state_tax_expire="2017-05-20"
             state_tax_status="1"
             name="Name"
             phone="12345"
             fax="12345"
             email="test@email.com"
             address_one="Address 1"
             address_two="Address 2"
             city="City"
             state="State"
             zipcode="00000"
             status="1" />
	
    <agency_type id="100"
                 name="Name"
                 status="1" />

    <agency id="100"
            license_no="License"
            company_id="100"
            company_id1="100"
            agency_type_id="100"
            name="Name"
            tracking_mode="1"
            contact_person="Contact Person"
            email="test@email.com"
            address_one="Address 1"
            address_two="Address 2"
            city="City"
            state="State"
            zipcode="00000"
            timezone="UTC"
            phone="12345"
            holiday="Holiday"
            fax="12345"
			status="1"/>

	<user id="100"
		  company_id="100"
		  agency_id="100"
		  user_type="1"
		  username="username"
		  password="password"
		  first_name="First Name"
		  last_name="Last Name"
		  phone="12345"
		  medicaid_no="Medicaid"
		  insurance_start="2017-05-01"
		  insurance_end="2017-05-01"
		  insurance_eligiable="Eligiable"
		  eligiable_start="2017-05-01"
		  eligiable_end="2017-05-01"
		  family_doctor="Family Doctor" 
		  status = "1"
		  status_second="1"/>
   
   <employee id="100"
			  first_name="first_name"
			  last_name="last_name"
			  gender="Male"
			  social_security_number="1234568"
			  date_of_birth="2017-05-01"
			  physical_exam="physical exam"
			  certificate_name="certificate name"
			  certificate_start="2017-05-01"
			  certificate_end="2017-06-01"
			  work_start="2017-05-01 09:00:000"
			  work_end="2017-06-01 21:00:000"
			  position="emp"
			  manager="No"
			  type="full time"
			  status="1"
			  background_check="pass"
			  created_at="2017-05-01"
			  updated_at="2017-05-01"
			  agency_id="100"
			  active="1"/>
	         
    <serviceplan id="100"
		    	approvedby = "approver"
		    	employee_id = "100"
		    	plan_start = "2017-05-01"
		    	plan_end = "2017-06-01"
		    	days = "10"
		    	docurl = "/test/restdoc"
		    	created_at = "2017-06-01"
		    	updated_at = "2017-06-01"
		    	user_id= "100"/>     
    
	<visit id="100" 
			agency_id="100" 
			selected_seat="1" 
			user_id="100"
			status="Status"
			active="1"/>
	
	<activity id="100"
			  name = "activity name"	
			  status = "1"
			  instructor_employee_id = "100"
			  />			
	   
	  <visit_has_activity
	  		visit_id = "100"
	  		activity_id = "100"
	  		table_name = "table name"
	  /> 
</dataset>