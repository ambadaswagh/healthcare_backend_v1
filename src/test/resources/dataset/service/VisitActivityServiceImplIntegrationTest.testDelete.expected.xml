<?xml version="1.0" encoding="UTF-8"?>
<dataset>

    <visit_has_activity />

</dataset>